crates/sui-types/src/event.rs

```diff
@@ -157,10 +157,27 @@ impl Event {
             contents: vec![],
         }
     }
+
+    /// A substitute event for a system epoch info event,
```

Comment:
this is not used in this pr, maybe useful if we want to emit a sub. event from FN, okay to remove this too.